test/Generics/unbound.swift

```diff
@@ -137,3 +137,19 @@ do {
   assertExactType(of: x1, is: Generic<Int, Bool>.self)
   assertExactType(of: x2, is: Generic<Int, Bool>.self)
 }
+
+// https://github.com/apple/swift/issues/60922
+enum T<E> {
```

Comment:
Could you transpose the names of the enum and generic parameter to make it feel more natural?